.gitignore

```diff
@@ -49,3 +49,35 @@ hs_err_pid*
 # data which is too big to put into git
 data/On_Time_On_Time*
 data/2016_*.csv
+data/readme.html
+
+# Created by https://www.gitignore.io/api/macos
+
+### macOS ###
```

Comment:
Mostly the .DS_Store files. But I figured I'd add other known files as well. I'll change it to just *.DS_Store.